### ISSUE TITLE
[Accessibility]: Upload state image doesn't have suitable alt text

### ISSUE THREAD

**Issue by Strum355** [sourcegraph/sourcegraph#35330]
### Audit type

Screen reader navigation

### User journey audit issue

https://github.com/sourcegraph/sourcegraph/issues/33512

### Problem description

Images that preceed state describing text are announced only as "image" instead of giving a more apt description.

### Expected behavior

Provide alt text for the image instead of just "image" to indicate the state

### Additional details

![image](https://user-images.githubusercontent.com/18282288/167940778-ab164cdc-0dc7-48c6-9bec-11bacce35aee.png)

### Assigning labels

- [X] Please give this issue an estimate by applying a label like `estimate/Xd`, where X is the estimated number of days it will take to complete.
- [X] If this issue is specific to a specific Sourcegraph product, please assign the appropriate team label to this issue.
- [X] If this issue will require input from designers in order to complete, please assign the label `needs-design`.
- [X] If you are confident that this issue should be fixed by GitStart, please assign the label `gitstart`.

### Owner

This issue will be fixed by my team, I have assigned a relevant member to it, or I will do so in the near future.

**Comment by umpox** [comment]
Should now be fixed as part of https://github.com/sourcegraph/sourcegraph/issues/34582
